pkg/spec/cluster.go

```diff
@@ -193,6 +193,8 @@ type ClusterStatus struct {
 
 	// Size is the current size of the cluster
 	Size int `json:"size"`
+	// HealthySize is the number of pods passing the liveliness probe, HTTP GET on port 2379
```

Comment:
It's inconvenient to display `[]*Pod` in json.
It's preferred to use `healthyEndpoints []string`.
Since it's endpoints, better be `ip/hostname:port`.